src/lib/debounce.js

```diff
@@ -1,9 +0,0 @@
-export default function debounce(fn, time) {
```

Comment:
could probably use this as a fallback for browsers that don't have rAF, so maybe keep it around.